DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/Mocks/MockForEnumerableQuery.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Data.Entity.Infrastructure;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitHubExtension.Security.Tests.Mocks
+{
+    class MockForEnumerableQuery<T> : EnumerableQuery<T>, IDbAsyncQueryProvider
```

Comment:
When I use EnumerableQuery a FirstOrDefaultAsync() throws an exception:  "The provider for the source IQueryable doesn't implement IDbAsyncQueryProvider. Only providers that implement IDbAsyncQueryProvider can be used for Entity Framework asynchronous operations."